src/providers/apolloProvider.ts

```diff
@@ -153,6 +153,9 @@ export class ArtificialApollo {
 
   private throwError = debounce((error: any) => {
     this.outputLog.log(`Problem connecting to Artificial, check token/config ${error} ${error.networkError.result}`);
+    if (error.networkError.statusCode === 403) {
+      ConfigValues.getInstance().promptForToken();
+    }
```

Comment:
We need to get rid of this.  Codespaces save the file on every edit, which can trigger this multiple times when you're in the middle of fixing up your config/secrets.

Reply:
Debounced it down to only automatically prompting at most once every 30 seconds. Do you think that would be enough time? Setting the instance config is also going to be much easier with the exclusion of the prefix and org.